pandas/tests/frame/test_operators.py

```diff
@@ -1161,6 +1161,33 @@ def test_inplace_ops_identity(self):
         assert_frame_equal(df2, expected)
         assert df._data is df2._data
 
+        # no id change
+        df = DataFrame({'a': [1., 2., 3.]})
+        expected = id(df)
+        df += 2.
```

Comment:
can you make this a new test, ideally you can parametrize this and also test (in addition to identity) equality with the non-inplace op
e.g. something like
```
@pytest.mark.parmatrize('op', ['add', 'sub'......])
def test_inplace_ops_identity2(self, op):
       df = DataFrame(...)
       df_copy = df.copy()
       iop = '__i{}__'.format(op)
       op = '__{}__'.format(op)
       
       getattr(df, iop)(2.)
       expected = getattr(df_copy, op)(2.)
       tm.assert_frame_equal(df, expected)
       ....check id here too
```